### ISSUE TITLE
Missing thumbnail images

### ISSUE THREAD

**Issue by lilHermit** [harlan-zw/unlighthouse#245]
### Describe the bug

I'm seeing missing thumbnails when executing inside docker

As seen below this site has 2 manualled added urls, as it scans the 404s for the thumbnails appear and clicking each shows the correct full size image

![image](https://github.com/user-attachments/assets/0bea1d2b-35a0-4b8b-afe7-7373d37cfdbb)

### Reproduction
```bash
docker run -it --rm -w /app -v $(pwd):/app -p 5678:5678 --shm-size=2g node:18 bash
## inside the container

## Setup
npm install unlighthouse
npx puppeteer browsers install chrome
apt update && apt install gconf-service libasound2 libatk1.0-0 libatk-bridge2.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget

## Execute
npx unlighthouse --site https://example.com



### System / Nuxt Info

  System:
    OS: Linux 6.8 Debian GNU/Linux 12 (bookworm) 12 (bookworm)
    CPU: (16) x64 AMD Ryzen 7 5700G with Radeon Graphics
    Memory: 43.86 GB / 62.15 GB
    Container: Yes
    Shell: 5.2.15 - /bin/bash
  Binaries:
    Node: 18.20.3 - /usr/local/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 10.7.0 - /usr/local/bin/npm

**Comment by lilHermit** [comment]
Ok I've worked out what the problem is. I was running `accessibility` only with 

```
    lighthouseOptions: {
        onlyCategories: ['accessibility'],
    }
```

Removing this or adding `performance` in fixes the issue, so I assume the images are generated in that stage?!

**Comment by cy-josh** [comment]
I'm also having this issue when running the scan with only the `accessibility` category enabled + generating a static report.

I've noticed that the static files are looking for an image file called `screenshot.jpeg` but the static files only include an image file called `full-screenshot.jpeg` in the report directory for a specific page.

![image](https://github.com/user-attachments/assets/a86f725c-b125-4c0b-9aac-85129089927f)

![image](https://github.com/user-attachments/assets/2528ed8e-6747-467f-964f-6ff7e8fff63f)

**Comment by harlan-zw** [comment]
Hi, thanks for the issue, I've pushed up a fix in [v0.14.0](https://github.com/harlan-zw/unlighthouse/releases/tag/v0.14.0).
